.yarn/patches/@metamask-assets-controllers-npm-45.1.0-d914c453f0.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/dist/token-prices-service/codefi-v2.mjs b/dist/token-prices-service/codefi-v2.mjs
+index a13403446a2376d4d905a9ef733941798da89c88..19fd490a5d0446c29ad24cc39465f3984a969406 100644
+--- a/dist/token-prices-service/codefi-v2.mjs
++++ b/dist/token-prices-service/codefi-v2.mjs
+@@ -12,8 +12,7 @@ var __classPrivateFieldGet = (this && this.__classPrivateFieldGet) || function (
+ var _CodefiTokenPricesServiceV2_tokenPricePolicy;
+ import { handleFetch } from "@metamask/controller-utils";
+ import { hexToNumber } from "@metamask/utils";
+-import $cockatiel from "cockatiel";
+-const { circuitBreaker, ConsecutiveBreaker, ExponentialBackoff, handleAll, retry, wrap, CircuitState } = $cockatiel;
++import { circuitBreaker, ConsecutiveBreaker, ExponentialBackoff, handleAll, retry, wrap, CircuitState } from "cockatiel"
```

Comment:
This patch is needed for the  `storybook:build` command.
Check the other (deleted) patches. The remaining changes from the old patches are not required anymore.